api/RTCPeerConnection.json

```
@@ -1633,6 +1633,58 @@
             "deprecated": false
           }
         },
+        "Legacy_getStats": {
```

I don't think we have a rule about it, but I think it's unusual to include the parent feature name in the subfeature name (see my note below to understand this suggestion)

```suggestion
        "with_callbacks": {
```

---

This is fixed in the update I'm getting ready to submit.